dao/src/test/java/org/thingsboard/server/dao/sql/query/EntityDataAdapterTest.java

```diff
@@ -25,6 +25,7 @@ public class EntityDataAdapterTest {
     public void testConvertValue() {
         assertThat(EntityDataAdapter.convertValue("500")).isEqualTo("500");
         assertThat(EntityDataAdapter.convertValue("500D")).isEqualTo("500D"); //do not convert to Double !!!
-        assertThat(EntityDataAdapter.convertValue("0101010521130565")).isEqualTo("0101010521130565");
+        assertThat(EntityDataAdapter.convertValue("0101010521130565")).isEqualTo("0101010521130565"); //do not convert to Double !!!
```

Comment:
You have added check for "Simple double" that contained ".", but I don't see new tests values that contain ".". When do you return Double as double?

Reply:
`EntityDataAdapter.convertValue` value method always accepts strings as input parameter. 

Simple double added to verify that BigInteger value like `89010303310033979663` will not be converted to: 8.901030331003398E19. 

Please check the example added in the github issue: 

https://github.com/thingsboard/thingsboard/issues/8001

Reply:
@ShvaykaD, how about adding the exact test case for the issued value `89010303310033979663` ?